vmware/ngc/NGC-AdapterManager/src/main/java/org/opensds/vmware/ngc/adapters/opensds/Main.java

```diff
@@ -0,0 +1,20 @@
+package org.opensds.vmware.ngc.adapters.opensds;
+
+import java.util.List;
+
+import org.opensds.vmware.ngc.models.SnapshotMO;
+import org.opensds.vmware.ngc.models.VolumeMO;
+
+public class Main {
+	public static void main(String args[]) throws Exception {
+		OpenSDS openSDS = new OpenSDS("opensds");
+		openSDS.login("192.168.20.159", 50040, "admin", "opensds@123");
+		List<VolumeMO> list = openSDS.listVolumes();
+		//VolumeMO vol =openSDS.queryVolumeByID("688d5620cb544e49b8d759c0c517c16c1");
+		//System.out.println(vol.name);
+		for(VolumeMO volume : list) {
```

Comment:
remove unused code here

Reply:
Remove this file, hope this s for testing

Reply:
Done, Removed this file